gmm_client/test/basic_SUITE.erl

```diff
@@ -0,0 +1,102 @@
+-module(basic_SUITE).
+-include_lib("common_test/include/ct.hrl").
+-include_lib("stdlib/include/assert.hrl").
+-export([
+    all/0,
+    groups/0,
+    init_per_suite/1,
+    end_per_suite/1,
+    init_per_group/2,
+    end_per_group/2
+]).
+-export([
+    vertices_test/1,
+    edges_test/1
+]).
+
+groups() -> [{basic_flow, [sequence], [
+    vertices_test,
+    edges_test
+]}].
+
+all() -> [{group, basic_flow}].
+
+init_per_suite(Config) ->
+    [{ct_hooks, [docker_compose_cth]} | Config].
+
+end_per_suite(_Config) ->
+    ok = application:stop(gmm_client).
+
+init_per_group(basic_flow, _Config) ->
+    timer:sleep(1000), %% from unknown reasons some delay is needed before executing first tests - without it tests fail with reason {badmatch, {error, closed}}
+    {ok, _} = application:ensure_all_started(gmm_client),
+    ok;
+init_per_group(_Group, _Config) ->
+    ok.
+
+end_per_group(_Group, _Config) ->
+    ok.
+
+vertices_test(_Config)->
+    % given
+    client:add_user("Jan"),
+    client:add_group("Friends"),
+    client:add_space("Space"),
+    client:add_provider("Provider"),
```

Comment:
Więcej napisałem niżej, ale nawiązując: mogłaby tu być lista wierzchołków, które tworzysz:
[{type1, name1}, {type2, name2},..]

Ta lista byłaby modyfikowana potem kiedy modyfikujesz graf (np. usuwasz wierzchołek) i sprawdzanie poprawności to byłoby porównanie zawartości listy z tym, co zwraca listing z serwera.

Nie upieram się, ale do rozważenia

Reply:
rozważę to może w drugim pr, jak będę pokrywał testami resztę api